tests/conftest.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from .dash_core_components_page import DashCoreComponentsMixin
+from dash.testing.browser import Browser
+
+
+class DashCoreComponentsComposite(Browser, DashCoreComponentsMixin):
+    def __init__(self, server, **kwargs):
+        super(DashCoreComponentsComposite, self).__init__(**kwargs)
+        self.server = server
+
+    def start_server(self, app, **kwargs):
+        """start the local server with app"""
+
+        # start server with app and pass Dash arguments
+        self.server(app, **kwargs)
+
+        # set the default server_url, it implicitly call wait_for_page
+        self.server_url = self.server.url
+
+
+@pytest.fixture
+def dash_dcc(request, dash_thread_server, tmpdir):
```

Comment:
@alexcjohnson  all the dcc utils will be defined in `dash_core_components_page.py`